awx/job_template_launch_resource.go

```diff
@@ -76,18 +78,29 @@ func (r *JobTemplateLaunchGetResponse) JobTemplateData() *JobTemplate {
 type JobTemplateLaunchPostRequest struct {
 	Request
 
-	extraVars string
+	extraVars map[string]interface{}
 }
 
-func (r *JobTemplateLaunchPostRequest) ExtraVars(value string) *JobTemplateLaunchPostRequest {
+func (r *JobTemplateLaunchPostRequest) ExtraVars(value map[string]interface{}) *JobTemplateLaunchPostRequest {
 	r.extraVars = value
 	return r
 }
 
+func (r *JobTemplateLaunchPostRequest) ExtraVar(name string, value interface{}) *JobTemplateLaunchPostRequest {
+	r.extraVars[name] = value
+	return r
+}
+
 func (r *JobTemplateLaunchPostRequest) Send() (response *JobTemplateLaunchPostResponse, err error) {
 	// Generate the input data:
 	input := new(data.JobTemplateLaunchPostRequest)
-	input.ExtraVars = r.extraVars
+
+	// convert extravars json to string
+	bytes, err := json.Marshal(r.extraVars)
+	if err != nil {
+		return
+	}
+	input.ExtraVars = string(bytes)
```

Comment:
So this `input.ExtraVars` should also be a `map[string]interface{}`, if I understand correctly.

Reply:
I will check against awx

Reply:
@zgalor proved to me that I was wrong, this can be an string, sorry for the noise.

Reply:
I tested againts awx and string is parsed OK so I'm leaving as is